docs/l10n.rst

```diff
@@ -0,0 +1,148 @@
+committed see `When can I expect to see my strings land?`_ for details.
+
+The hosts for the new frontend are the same as addons-server except be routed
+to the new frontend you either need:
+
+    * A "mamo" cookie with the value "on". You can set this by clicking
+      "View mobile site" in the footer.
+    * A Mobile user-agent string.
+
+Discovery-Pane app
+++++++++++++++++++
+
+The addons-frontend repo is also home to the Discovery pane app (disco.po)
+which is the site containing curated addons available at about:addons
+(tools -> Add-ons -> Get Add-ons).
+
+Whilst this looks like it's part of firefox it's actually a webpage served
+from ``https://discovery.addons.mozilla.org``.
+
+Discovery Pane editorial content
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The content for the disco pane comes from the API so if you want to
+translate that you will need to look in addons-server.
+
+Testing Discovery Pane localizations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The strings will be available on -dev or stage depending on when they were
+committed see `When can I expect to see my strings land?`_ for details.
+
+The hosts for the discovery pane are as follows:
+
+    * dev ``https://discovery.addons-dev.allizom.org``
+    * stage ``https://discovery.addons.allizom.org``
+    * prod ``https://discovery.addons.mozilla.org``
+
+Advanced Configuration for testing addons-frontend strings
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This will be enough for basic string checks. If you need to test strings
+related to the install buttons you will need to enabled preferences to allow
+add-on installation to work on the development and stage servers.
+
+To fully configure Firefox to install add-ons on either dev/stage you will
+need to additionally do the following:
+
+    * Use nightly or dev edition
+    * A new profile is recommended since changing the dev certs will make
+      existing add-on installs as invalid.
+    * Go to ``about:config`` and set ``extensions.webapi.testing`` as a
+      Boolean to ``true``.
+    * To install an add-on, you will need to be able to install a signed
+      add-on from dev or stage. Create the ``xpinstall.signatures.dev-root``
+      pref in ``about:config`` as a Boolean and set it to ``true``.
+    * (Discovery Pane only) Go to about:config and change the host in
+      ``extensions.webservice.discoverURL`` to
+      ``https://discovery.addons-dev.allizom.org/`` for -dev
+      or ``https://discovery.addons.allizom.org/`` for stage.
+    * If you see warnings when trying to install a theme or theme previews
+      don't work go to ``about:preferences#security`` and add an exception
+      for dev/stage (See the "Exceptions" button next to "Warn me when
+      sites try to install add-ons").
+    * Restart the browser.
+
+
+Routine Extraction of new strings
+---------------------------------
+
+Newly added strings are extracted every week after the tag which is at 1700
```

Comment:
> I say we use a time zone that first off a lot of people already use (compared to UTC, which no one in all of the Mozilla org chart, to my knowledge, uses).

This could have been the beginning!

/me hides